sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,72 @@
 Version history
 ===============
 
+8.0.0 (2024 October)
+--------------------
+
+File format fixes and improvements:
+
+* CV7000
+   - allow TIFF files to be in a subdirectory
+
+* DICOM
+   - add support for reading precompressed tiles [1]_
+   - improve image write performance [1]_
+   - fix writing to DICOM when physical size units are missing from input data
+
+* Evident/Olympus OIR
+   - improve image read performance (thanks to Nicolas Chiaruttini)
+
+* Hamamatsu NDPI
+   - add support for reading precompressed tiles [1]_
+
+* Leica LIF
+   - fix file offset calculations for datasets with unstitched tiles
+
+* OME-TIFF
+   - improve image write performance
+   - add support for writing precompressed tiles [1]_
+
+* OME-XML
+   - fix incorrect logical operators when checking image dimensions
+
+* RCPNL
+   - define objective metadata for lens ID 18112
+
+* SDT
+   - fix reading of files acquired with SPCM 9.90 (thanks to Chen Qian)
```

Comment:
i assume Chen Qian will be added to the list of contributors on www.o.org

Reply:
Yes, that is done in https://github.com/ome/www.openmicroscopy.org/pull/757